geth/api/backend.go

```diff
@@ -94,7 +94,7 @@ func (b *StatusBackend) startNode(config *params.NodeConfig) (err error) {
 	if err != nil {
 		switch err.(type) {
 		case node.RPCClientError:
-			err = node.ErrRPCClient
+			err = fmt.Errorf("%v: %v", node.ErrRPCClient, err)
```

Comment:
Ah, better solution. But shows how an error package with annotate could help to introduce a culture of extending error informations.